### ISSUE TITLE
[FEA] classification example

### ISSUE THREAD

**Issue by subbayya** [NVIDIA-Merlin/NVTabular#1637]
I tried changing the loss function to BCELoss.  I got a message saying that
Many models use a sigmoid layer right before the binary cross entropy layer? How do I do that using the nvtabular regression example for pytorch?

**Comment by rnyak** [comment]
@subbayya can you please give us more info regarding which notebook you are running?  what did you change in your code? what's the error you are getting?

**Comment by rnyak** [comment]
@subbayya are you trying to run PyTorch classification example? did you test `torch.nn.BCEWithLogitsLoss()` instead?  you can feed it [here](https://github.com/NVIDIA-Merlin/NVTabular/blob/main/nvtabular/framework_utils/torch/utils.py#L64) as a `loss_func` arg. 

Besides we have several classification examples with [Merlin Models](https://github.com/NVIDIA-Merlin/models) using TF framework.
